app-android-xamarin/Moodis/Moodis/Feature/Menu/MenuActivity.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+using Android.App;
+using Android.Content;
+using Android.Graphics;
+using Android.OS;
+using Android.Support.V7.App;
+using Android.Util;
+using Android.Widget;
+using Java.Lang;
+using Moodis.Feature.CameraFeature;
+using Moodis.Ui;
+
+namespace Moodis.Feature.Menu
+{
+    [Activity(Label = "Menu Activity")]
+    public class MenuActivity : AppCompatActivity
+    {
+        private const string WarningInRequest = "Something wrong happened. Is your internet turned on?";
+        private const string WarningFaceDetection = "Face not detected, please try to use better lighting and stay in front of camera";
+        private const string WarningPlayingMusic = "Because face was not detected, cannot play music based on it.";
+        private readonly string TAG = nameof(MenuActivity);
+        public MenuViewModel ActivityMenuViewModel { get; set; }
+        private AppCompatActivity parentActivity;
+        private const string FormatDouble = "N3";
+
+        protected override void OnCreate(Bundle savedInstanceState)
+        {
+            base.OnCreate(savedInstanceState);
+            ActivityMenuViewModel = new MenuViewModel();
+            SetContentView(Resource.Layout.activity_menu);
+
+            Intent intent = Intent;
+            ActivityMenuViewModel.currentImage.ImagePath = intent.GetStringExtra("ImagePath");
+
+            InitButtonsAndInputs();
+            UpdateLabels();
+            DeleteImage(ActivityMenuViewModel.currentImage.ImagePath);
+        }
+
+        public async void UpdateLabels()
+        {
+            var imageBox = FindViewById<ImageView>(Resource.Id.imageForView);
+            Bitmap bmImg = BitmapFactory.DecodeFile(ActivityMenuViewModel.currentImage.ImagePath);
+            bmImg = RotateImage(bmImg);
+            imageBox.SetImageBitmap(bmImg);
+            var emotionLabels = new List<TextView> { FindViewById<TextView>(Resource.Id.lblAnger), FindViewById<TextView>(Resource.Id.lblContempt), FindViewById<TextView>(Resource.Id.lblDisgust),
+                FindViewById<TextView>(Resource.Id.lblFear), FindViewById<TextView>(Resource.Id.lblHappiness), FindViewById<TextView>(Resource.Id.lblNeutral), FindViewById<TextView>(Resource.Id.lblSadness),
+                FindViewById<TextView>(Resource.Id.lblSurprise) };
+            foreach (var label in emotionLabels)
+            {
+                label.Text = "loading";
```

Comment:
move this string to strings.xml and fetch it using `this.GetString(R.string.loading)`. Same applies to other constant string values